docs/csharp/language-reference/compiler-messages/nullable-warnings.md

```diff
@@ -215,6 +215,32 @@ The following warnings indicate that you haven't set the nullable context correc
 - **CS8636** - *Invalid option for `/nullable`; must be `disable`, `enable`, `warnings` or `annotations`*
 - **CS8637** - *Expected `enable`, `disable`, or `restore`*
 
+To set the nullable context correctly, you have two options:
+
+1. **Project-level configuration**: Add the [`<Nullable>`](../compiler-options/language.md#nullable) element to your project file:
+
+  ```xml
+  <PropertyGroup>
+      <Nullable>enable</Nullable>
+  </PropertyGroup>
+  ```
+
```

Comment:
Looks like the blank lines need to be removed:

````suggestion
  ```xml
  <PropertyGroup>
      <Nullable>enable</Nullable>
  </PropertyGroup>
  ```
````